### ISSUE TITLE
Marker Support

### ISSUE THREAD

**Issue by bjschwarz** [alexcode/vue2vis#56]
From what I can tell changes were added in the 0.0.17 release to support customTimeMarkers. But I don't think markers where added in the visjs library until after 4.21.0. 

@alexcode do you know if 0.0.18 should support markers or will this not be until #52 has been addressed?

**Comment by alexcode** [comment]
Yes, I'm not planning to add more functionality before the refactoring I'm working on.
